MetaMask/metamask-extension
# Update `electron` from 11.4.12 to 11.5.0

This should include no functional changes. We only use `electron` for `react-devtools`, which appears to still work after this update.

This replaces PR #12338.

Manual testing steps:  
  - Create a dev build, then stop the watch process when the build has finished.
  - Run `yarn devtools:react`.
 - Install the dev build and ensure that the React dev tools still work correctly.
